build.gradle.kts

```diff
@@ -32,9 +34,18 @@ apply(from = rootProject.file("dependencies.gradle.kts"))
 apply(from = "scripts/git-hooks/install.gradle")
 
 measureBuilds {
-    enable = project.extra.get("measureBuildsEnabled")?.toString().toBoolean()
-    automatticProject = MeasureBuildsExtension.AutomatticProject.PocketCasts
-    authToken = project.extra.get("appsMetricsToken")?.toString()
+    enable = project.extra.properties.get("measureBuildsEnabled")?.toString().toBoolean()
+    onBuildMetricsReadyListener {
+        val report = this@onBuildMetricsReadyListener
```

Comment:
Minor (🔍): The [docs](https://github.com/Automattic/measure-builds-gradle-plugin?tab=readme-ov-file#setup) specify that instead of during this you could do something like this: `onBuildMetricsReadyListener { MetricsReport metricsReport -> ... }`. But, this seems not be working as expected, right? Maybe we should update the docs. Or, if that is indeed somehow working, maybe update the usage here. Wdyt? 🤔

Reply:
The docs are referencing `*.gradle` file - this is just a difference between Groovy and Kotlin syntax. The syntax from docs works e.g. here: https://github.com/bloom/DayOne-Android/pull/4319/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R54-R61 🙂

Reply:
Ah yes, right, true, thanks for the clarification on that Wojtek... 😅